Solutions/TheHive/Package/mainTemplate.json

```diff
@@ -33,515 +33,9 @@
     "solutionId": "azuresentinel.azure-sentinel-solution-thehive",
     "_solutionId": "[variables('solutionId')]",
     "email": "support@microsoft.com",
```

Comment:
@v-amolpatil : this file is not relevant, please delete it from this PR

Reply:
done